runhouse/servers/autostop_servlet.py

```diff
@@ -0,0 +1,32 @@
+import time
+
+
+class AutostopServlet:
+    """A helper class strictly to run SkyPilot methods on OnDemandClusters inside SkyPilot's conda env."""
+
+    def __init__(self):
+        self._last_activity = time.time()
+        self._last_register = None
```

Comment:
I don't think we need the last_registered last_activity stuff anymore. We can just set the activity directly in SkyPilot, there's no reason to set it lazily like we did before.

Reply:
Nah currently we also do `set_last_active_time_to_now` in `get_env_servlet_name_for_key`. This happens quite often, we don't need to re-set it everytime, and the loop is always running, so we can just do it there.